### ISSUE TITLE
Did you see this error reported on r-pkg-devel?

### ISSUE THREAD

**Issue by bryanhanson** [zeehio/MassSpecWavelet#7]
I saw this and it looks like nobody has opened an issue on it.  I can't find a direct link to the conversations, so attaching a screenshot of one of the later exchanges.

![Screenshot 2024-12-17 at 9 07 35 PM](https://github.com/user-attachments/assets/3021439d-c3fb-48ea-bb72-4cfdd5ab5c9f)

**Comment by zeehio** [comment]
Hi Bryan, thanks for reaching out.

Vincent Carey forwarded me that thread a couple of days ago. Unfortunately I am subscribed to r-devel but not r-pkg-devel so I don't think I can easily reply.

I have fixed the issue on Bioconductor devel, and it has already been published at Bioconductor devel  as MassSpecWavelet 1.73.1.

I also pushed a fix to the stable Bioconductor release. I am still waiting to see the stable version published. I've waited for 39 hours, so if in 9 hours I don't see it published I will reach out to Bioconductor maintainers. It should be released soon as MassSpecWavelet 1.72.1 on Bioconductor 3.20.

I wish I had seen the issue earlier
